src/modules/transaction/components/TxComposer/index.js

```diff
@@ -20,7 +21,7 @@ const TxComposer = ({
   children, transaction, onComposed, onConfirm, className, buttonTitle, transactionData,
 }) => {
   const { t } = useTranslation();
-  const network = useSelector(state => state.network);
+  useSchemas({ chainId: 1 });
```

Comment:
why do we need to pass chainid? since the selected blockchain application URL will be present already we just need to all `http://BLOCHAIN_APP_URL/api/v3/commands/parameters/schemas`, which is handled in query handler?